odincal/Dockerfile

```diff
@@ -18,7 +18,10 @@ RUN pip install -r requirements.txt
 
 # Install oops and odincal
 COPY ./oops /oops
+WORKDIR /oops/oops
+RUN gcc -shared -o libfft.so -fPIC ../Library/odinfft.c
```

Comment:
What's the effect of this?

Reply:
Since there is no github action to deploy yet, I guess you already have the lib pre compiled in your working repo?